lib/common/util/constants.js

```diff
@@ -2140,7 +2148,14 @@ var Constants = {
     }
   },
 
-  CompatibleVersionConstants: {    
+  CompatibleVersionConstants: {
+    /**
+    * Constant for the 2015-12-11 version.
+    *
+    * @const
+    * @type {string}
+    */
+    DEC_2015: '2015-12-11',
```

Comment:
Only AUGUST_2013 and FEBRUARY_2012 are allowed in this constants. We don't have compatibility support in other service version and we requires the SAS token version should always be identical with the current API version in the SDK.